src/solc_0.7/common/BaseWithStorage/ERC20ExecuteExtension.sol

```diff
@@ -0,0 +1,179 @@
+pragma solidity 0.7.1;
+
```

Comment:
We do not need that file.
ExecuteExtension is deprecated. this is only used for SAND